### ISSUE TITLE
Post Kinds disables my Post Types in the adminscreen

### ISSUE THREAD

**Issue by frankmeeuwsen** [dshanske/indieweb-post-kinds#297]
I noticed recently the metabox with Post Types was missing in my admin screen. It is not disabled in the screen settings and I still need it sometimes to check post types of specific posts. When I disable the Post Kinds plugin it shows up again. When Post Kinds is enabled it dissapears.
I run version 3.3.3 on my live server, locally I run 3.2.6 (Haven't updated yet) where the WordPress Post Types are still shown. 
I found out that some of my posts are saved as WP Post Types "Asides" while I put the Post Kind on article. This causes my theme to show unexpected behaviour. I am not quite sure how I posted these articles, must be through some Micropub client but I can't find out which one. 
My standard Post Type format in the settings of the live server is "Standard"
So is there a way to make the Post Types visible again in the admin while also use Post Kinds?

**Comment by dshanske** [comment]
Do you mean post formats? Post Kinds replaces post formats, that's why it disables it.

**Comment by frankmeeuwsen** [comment]
Yes. But WordPress still uses the post formats to declare classes (e.g. post-aside) and when I want to make a specific template for notes, I need to call it content-aside.php. So while it's disabled, it's still used by the theme. So I'm still quite unsure if the post formats should be disabled at all. Because in earlier versions of the plugin, post formats were enabled. Why is this changed then?

**Comment by ndegruchy** [comment]
I'm bumping this ticket because when I install the Post Kinds plugin, it hides anything that doesn't have a `post-kind` set. So, items that I may have tagged with a `post-format` are all hidden until I mark it with a `post-kind`.

I'm not sure why this plugin would bother filtering out this data, but it's certainly not what I expected when I installed it.

**Comment by dshanske** [comment]
Because it is noted as a replacement for Post Formats. Let me see if I can add an override for you.

**Comment by ndegruchy** [comment]
> Because it is noted as a replacement for Post Formats. Let me see if I can add an override for you.

I mean, I guess I get it. It was just a surprise seeing my front page go from 100+ posts to 0. I experimented by marking one of them with a `post-kind` and it shows up.

Maybe a checkbox to not filter anything from the front page? I mean, I can also go back and mark everything appropriately, but that is going to be a lot of effort. I'm _trying_ to be more "indieweb", but the options for WordPress are confusing, especially trying to migrate to such a state.

**Comment by dshanske** [comment]
That was my plan.. I want to make sure what you described doesn't happen.

**Comment by dshanske** [comment]
Also, anything I can help with... reach out

**Comment by ndegruchy** [comment]
Thanks, David. I'll certainly do that.

**Comment by dshanske** [comment]
@ndegruchy Posts with no kind will now show on the main page.
